app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:email])
+    if user && user.authenticate(params[:password])
+      # ユーザーログイン後にユーザー情報のページにリダイレクトする
+      session[:user_id] = user.id
+      redirect_to redirect_to user_path(user.id)
```

Comment:
`redirect_to`2回実装してないかな？
```suggestion
     redirect_to user_path(user.id)
```